src/flint/fmpz.jl

```diff
@@ -1360,7 +1360,7 @@ end
 @doc Markdown.doc"""
     eulerphi(x::fmpz)
 > Return the value of the Euler phi function at $x$, i.e. the number of
-> positive integers less than $x$ that are coprime with $x$.
+> positive integers up to $x$ (inclusive) that are coprime with $x$.
```

Comment:
It seems a bit weird to say "inclusive" as usually it doesn't matter... is it to take the case `x == 1` into account? Anyway, I guess I would slightly prefer "less than or equal to", but it's fine with "up to `x` (inclusive)" too.

Reply:
Yes, that's why. I don't mind if you want to change it to "less than or equal to" before merging. I only care that what was written was incorrect.

Reply:
Ah sorry, I had missed your example in the OP!